config/rbac/rbac.yaml

```diff
@@ -19,19 +19,55 @@ metadata:
   namespace: cartographer-system
 
 ---
-#!
-#!    TODO make use of granular roles rather than a catch-all "cluster-admin"
-#!         grants it more privileges than it needs.
-#!
 apiVersion: rbac.authorization.k8s.io/v1
 kind: ClusterRoleBinding
 metadata:
   name: cartographer-cluster-admin
 roleRef:
   apiGroup: rbac.authorization.k8s.io
   kind: ClusterRole
-  name: cluster-admin
+  name: cartographer-controller-admin
 subjects:
   - kind: ServiceAccount
     name: cartographer-controller
     namespace: cartographer-system
+
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: cartographer-controller-admin
+rules:
+  - apiGroups:
+      - carto.run
+    resources:
+      - workloads
+      - workloads/status
+      - clustersupplychains/status
+      - runnables/status
+    verbs:
+      - get
```

Comment:
is this effectively the same as `'*'`? sounds like we're listing all the verbs 😁

Reply:
With the new changes it is no longer the same- we removed get, list, and watch as they are covered by the other group.